### DISCUSSION TITLE
Any API to add access permissions for a specific repo for a Github app?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#24476] [Apps, API and Webhooks]
<p>I have an installed Github app in my organization for which I allow access to only a subset of the repos.   I’d like to use an API to programmatically add those repo access permissions for that app.  Is that possible?  If not, is it planned?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>HI <a class="mention" href="/gscott">@gscott</a>,</p>
<p>Thanks for being here! When you create a GitHub App, you set the <a href="https://developer.github.com/apps/managing-github-apps/editing-a-github-app-s-permissions/#editing-a-github-apps-permissions" rel="nofollow noopener">permissions</a> that define the resources the app can access via the REST API v3. I hope this helps!</p>
